x-pack/plugins/observability_solution/apm/server/routes/errors/get_error_groups/get_error_group_sample_ids.ts

```diff
@@ -66,16 +66,16 @@ export async function getErrorGroupSampleIds({
           should: [{ term: { [TRANSACTION_SAMPLED]: true } }], // prefer error samples with related transactions
         },
       },
-      _source: [ERROR_ID, 'transaction'],
+      fields: [ERROR_ID, 'transaction'],
```

Comment:
this should be `transaction*`?

Reply:
I forgot to revert that to use source as we will work on the errors in a separate issue